tests/test_packagehandler.py

```diff
@@ -344,14 +344,13 @@ def test_publish_tags_if_tag_needed_and_version_not_in_index(
 
     packagehandler.devpihandler.latest_pypi_version.return_value = pypiver
     packagehandler.publish('index1', 'index2', tag_if_needed=tag_if_needed)
-    tag_called = True if (
-        pypiver != packagehandler.version and tag_if_needed) else False
+    tag_called = pypiver != packagehandler.version and tag_if_needed
     assert packagehandler.githandler.tag_release.call_count == (
         1 if tag_called else 0)
 
 
 @pytest.mark.parametrize('pypiver', ['0.1', '1.0'])
 def test_is_version_in_index(packagehandler, pypiver):
     packagehandler.devpihandler.latest_pypi_version.return_value = pypiver
-    assert packagehandler._is_version_in_index('index') == (
-        True if pypiver == packagehandler.version else False)
+    bool_value = pypiver == packagehandler.version
```

Comment:
Readability: `bool_value` could be more descriptive e.g. `expected_is_version_in_index`